package.json

```diff
@@ -45,11 +46,6 @@
     "gulp-nsp": "^2.1.0"
   },
   "repository": "mateusmedeiros/generator-prelude",
-  "scripts": {
-    "link": "npm run prepublish && sudo npm link",
-    "prepublish": "cp -R generators generatorsSrc && gulp publish",
```

Comment:
Because node has a lot of support for the older stuff already. Also because there isn't a lot of es6 yet. Converting the codebase to es6+ is in the TO-DO list on the README.md.